bot/utils/blurple_formatter.py

```diff
@@ -0,0 +1,711 @@
+
+        return (
+            f"from{space(1)}{'.'*level}{module}{space(1)}import{space(1)}"
+            + f",{space()}".join(
+                f"{name.name}"
+                + (f"{space(1)}as{space(1)}{name.asname}" if name.asname else "")
+                for name in names
+            )
+            + f"{space()};"
+        )
+    if isinstance(node, ast.JoinedStr):
+
+        return ast.unparse(node)
+    if isinstance(node, ast.Lambda):
+        args, body = node.args, node.body
+
+        return f"lambda{space(1)}{unparse(args)}{space(1)}:{space()}{unparse(body)}"
+    if isinstance(node, ast.List):
+        elts = node.elts
+
+        return (
+            f"[{space()}"
+            + f",{space()}".join(unparse(elt, True) for elt in elts)
+            + f"{space()}]"
+        )
+    if isinstance(node, ast.ListComp):
+        elt, generators = node.elt, node.generators
+
+        return (
+            f"[{space()}"
+            + f"{unparse(elt)} "
+            + f"{space(1)}".join(map(unparse, generators))
+            + f"{space()}]"
+        )
+    if isinstance(node, ast.Module):
+        body = node.body
+
+        return "\n".join(indent(body, 0))
+    if isinstance(node, ast.Name):
+        id_ = node.id
+
+        return id_
+    if isinstance(node, ast.NamedExpr):
+        target, value = node.target, node.value
+
+        return f"({unparse(target)}{space()}:={space()}{unparse(value)})"
+    if isinstance(node, ast.Nonlocal):
+        names = node.names
+
+        return f"nonlocal{space(1)}{f',{space()}'.join(map(unparse, names))};"
+    if isinstance(node, ast.Pass):
+
+        return f"pass{space()};"
+    if isinstance(node, ast.Raise):
+        exc, cause = node.exc, node.cause
+
+        return (
+            f"raise{space(1)}{unparse(exc)}"
+            + (f"{space(1)}from{space(1)}{unparse(cause)}" if cause else "")
+            + f"{space()};"
+        )
+    if isinstance(node, ast.Return):
+        value = node.value
+
+        return f"return{space(1)}{unparse(value)};"
+    if isinstance(node, ast.Set):
+        elts = node.elts
+
+        return (
+            "{"
+            + space()
+            + ", ".join(unparse(elt, True) for elt in elts)
+            + space()
+            + "}"
+        )
+    if isinstance(node, ast.SetComp):
+        elt, generators = node.elt, node.generators
+
+        return (
+            "{"
+            + space()
+            + f"{unparse(elt)}{space(1)}"
+            + f"{space(1)}".join(map(unparse, generators))
+            + space()
+            + "}"
+        )
+    if isinstance(node, ast.Slice):
+        lower, upper, step = node.lower, node.upper, node.step
+
+        s = f"{lower or ''}{space()}:{space()}{upper or ''}{space()}:{space()}{step or ''}"
+        return ":" if s.replace(" ", "") == "::" else s
+    if isinstance(node, ast.Starred):
+        value = node.value
+
+        return f"*{space()}{parenthesize(value)}"
```

Comment:
What if the expression (`value`) doesn't need parenthese - is there something we can do about that?